2021-offseason/src/main/java/frc/robot/subsystems/Hood.java

```diff
@@ -0,0 +1,90 @@
+package frc.robot.subsystems;
+
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.ControlType;
+import com.team254.lib.util.InterpolatingDouble;
+import com.team254.lib.util.InterpolatingTreeMap;
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANPIDController;
+import com.revrobotics.ControlType;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.PIDSubsystem;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import org.photonvision.PhotonCamera;
+
+public class Hood extends SubsystemBase {
+    private final CANSparkMax hoodMain;
+    private final CANEncoder hoodEncoder;
+    private final CANPIDController hoodController;
+    private InterpolatingTreeMap closeTreeMap;
+    private InterpolatingTreeMap farTreeMap;
+    private PhotonCamera camera;
+    private double curdeg = 0;
```

Comment:
at least camel case this variable name so its easy to tell what it is just from the name

Reply:
which variable?